library_generation/test/resources/integration/java-bigtable/pr-description-golden.txt

```diff
@@ -0,0 +1,17 @@
+This pull request is generated with proto changes between googleapis commit 679060c64136e85b52838f53cfe612ce51e60d1d and fc3043ebe12fb6bc1729c175e1526c859ce751d8.
+
+Qualified commits are:
+
+[googleapis/googleapis@fbcfef0](https://github.com/googleapis/googleapis/commit/fbcfef09510b842774530989889ed1584a8b5acb)
+
+[googleapis/googleapis@63d2a60](https://github.com/googleapis/googleapis/commit/63d2a60056ad5b156c05c7fb13138fc886c3b739)
+
+BEGIN_COMMIT_OVERRIDE
+
+fix: [bigtable] extend timeouts for deleting snapshots, backups and tables
+
+chore: [bigtable] update retry settings for backup rpcs
```

Comment:
`chore` PRs should not be included, but I'm not sure if release-please is going to filter out it automatically in release-notes, if `chore` PRs are included in between `BEGIN_COMMIT_OVERRIDE` and `END_COMMIT_OVERRIDE`. Do you know @suztomo ? If not, we can ask it in the release automation chat.

Reply:
Asked this question in release automation chat, `chore` will not appear in release note unless explicitly configured to do so.